rust/sealevel/environments/mainnet3/chain-config.json

```diff
@@ -391,6 +391,9 @@
     },
     "protocol": "ethereum",
     "rpcUrls": [
+      {
+        "http": "https://still-red-moon.bsc.quiknode.pro/be390591d09c000af5f23b40a06b326ba2683a2d"
```

Comment:
is adding this url intended? If not, should remove from here and from the PR branch history

Reply:
Not intentional, ty